tests/test__estimators.py

```diff
@@ -77,6 +77,10 @@ def test__scale_nearest_neighbor_distance(
 @pytest.mark.parametrize('X, n_dims, error', [
     (np.random.uniform(size=(10, 9)), 1, None),
     (np.random.uniform(size=(10, 9)), 3, None),
+    (np.random.uniform(size=(10, 9)), [3, 3, 3], None),
```

Comment:
I guess it would be great to add one passing test for a new split, e.g.` [4, 3, 2]`. And to check for a passing numpy array, e.g. `np.array([4, 3,2])`